src/popup/components/TransferReview.vue

```diff
@@ -129,6 +129,10 @@ export default {
       'tippingV1',
       'tippingV2',
     ]),
+    isAex9() {
```

Comment:
I suggest to use more informative naming here. Like for example `isSelectedAssetAex9`.